src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -1424,7 +1475,26 @@ namespace Google.Example.Library.V1.Samples
                 SeriesString = "xyz3141592654",
             };
             PublishSeriesResponse response = await libraryServiceClient.PublishSeriesAsync(shelf, books, edition, seriesUuid);
-            // FIXME: inspect the results
+            // % % % output handling % % % %
```

Comment:
Is this meant to be here?

Reply:
Yes, this is testing code too. They are generated from: https://github.com/googleapis/gapic-generator/blob/2508ba81c8c2eea22b5a8b672f10f1c3ffb08030/src/test/java/com/google/api/codegen/testsrc/common/library_gapic.yaml#L292

Reply:
Thanks.